### DISCUSSION TITLE
Filter function when discover repositories

### DISCUSSION THREAD

**Discussion by hupo376787** [community/community#22895] [New to GitHub]
<p>I think it’s better to add a filter funtion when discover repositories.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/c0935eba70c01d0044b821ede487fd9336b11bb5.jpeg" data-download-href="/uploads/short-url/rtBpCcDvTOkYGaUhgtvHibL7ubX.jpg?dl=1" title="微信截图_20180730094405.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/14960168/181073663-f968fd9a-e63e-4599-bfa6-cd60f3b7c57f.jpg" alt="微信截图_20180730094405.jpg" data-base62-sha1="rtBpCcDvTOkYGaUhgtvHibL7ubX" width="493" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/c0935eba70c01d0044b821ede487fd9336b11bb5_2_493x500.jpg, https://aws1.discourse-cdn.com/github/optimized/1X/c0935eba70c01d0044b821ede487fd9336b11bb5_2_739x750.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/c0935eba70c01d0044b821ede487fd9336b11bb5_2_986x1000.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/c0935eba70c01d0044b821ede487fd9336b11bb5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">微信截图_20180730094405.jpg</span><span class="informations">994×1008 96.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/hupo376787">@hupo376787</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our community organized so users can more easily find information.</p>
<p>As you’ll notice, your topic is now here in the How to use Git and GitHub board.</p>
<p>Additionally, we’re always working to improve GitHub, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Cheers!</p>
